### DISCUSSION TITLE
More than expected argument present

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2970] [Q&A]
![Screenshot (90)](https://github.com/user-attachments/assets/6b6352ef-644f-47df-8fd6-02cd42866ffd)

**Comment by AlesxanDer1102** [comment]
This error occurs because the ERC20Mock contract has been updated in the new version of OpenZeppelin. You need to use the older version or create a mock similar to the old version, like the one in the repository https://github.com/Cyfrin/foundry-defi-stablecoin-cu/blob/main/test/mocks/ERC20Mock.sol

**Comment by EngrPips** [comment]
Hello @prakharmishra3956, The chronological-update section of this GitHub repo https://github.com/Cyfrin/foundry-full-course-cu/blob/main/chronological-updates.md touched on this issue and gave a solution. Always make sure you check `chronological-update` whenever you run into an issue where what used to work stopped working to see if the issue has been addressed.

![Screenshot 2024-12-04 at 08 27 34](https://github.com/user-attachments/assets/cad5494c-d3c6-459e-a16d-fac03bc83a98)

**Reply by prakharmishra3956** [reply]
It worked but now I am getting multiple errors.
![Screenshot (91)](https://github.com/user-attachments/assets/f5604e98-7204-4be6-b914-ebd7080ee4f5)

**Reply by EngrPips** [reply]
You can use `command + p` [Mac] or `cltr + p` [Windows], then type the name of the file you want to import in the search bar that appears, and like that, you can get the correct path for the file.
